### DISCUSSION TITLE
Install on K3S (Kubernetes) with ARM64 arch

### DISCUSSION THREAD

**Discussion by carlosV2** [requarks/wiki#5406] [Error / Bug Report]
Hi,

I'm trying to install wiki.js in a K3S on top of ARM64 architecture.
According to [this page in the wiki](https://docs.requarks.io/install/docker#arm-images), wiki.js should support ARM64 plus I can easily find [ARM64-supported images of PostgreSQL](https://hub.docker.com/_/postgres), so, in theory, the full app should be deployable in such architecture.

I've followed the instructions in the [kubernetes installation guide page](https://docs.requarks.io/install/kubernetes) but, despite the installation seems to work, upon inspection, the postgres container fails:
```sh
$ kubectl get pods
NAME                       READY   STATUS              RESTARTS   AGE
wikijs-7f6c8b9f54-lz55k    0/1     ContainerCreating   0          3s
wikijs-postgresql-0        0/1     Error               0          3s
```

If I print the logs for this container, I get the following:
```sh
$ kubectl logs wikijs-postgresql-0
standard_init_linux.go:228: exec user process caused: exec format error
```

I'm pretty sure this is because it is trying to run a non-ARM64 executable.
If I inspect the container deployed, I can see the image comes from `docker.io/bitnami/postgresql:11.5.0-debian-9-r60` which doesn't seem to [support the ARM64 architecture](https://hub.docker.com/r/bitnami/postgresql).

Is there a way to instruct helm to use an ARM64-supported image for postgresql? Am I, perhaps, doing something wrong?

Many thanks.

**Comment by NGPixel** [comment]
The docker images are always built for AMD64, ARM64 and ARMv7.

However, the current helm definition might not properly support ARM64. I'm not knowledgeable enough to look into it however. Most of the helm files were contributed by the community. If you know your way around, you could probably edit the files to make it work: https://github.com/requarks/wiki/tree/main/dev/helm

**Comment by lt-kraken** [comment]
Running into the same issues sadly, did almost the same as you have. Tried modifying the StatefulSet for Progressql to several images that were build for ARM64, but to no avail. 

Slowly giving up on WikiJS, wasting too much time atm.
